client/src/components/About.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div>
+      <header>About</header>
+      <div>
```

Comment:
It's generally good to use as little HTML as possible, so since this `div` is unnecessary we can remove it.